cpu/esp32/periph/rtc.c

```diff
@@ -178,7 +179,7 @@ int rtc_set_alarm(struct tm *time, rtc_alarm_cb_t cb, void *arg)
      * system timer for alarms. The Advantage of using RTC over sytem timer
      * is that it also continues in deep sleep and after software reset.
      */
-    #if 0 /* TODO should be RTC_TIMER_USED */
+#if 0 /* TODO should be MODULE_ESP_RTC_TIMER */
```

Comment:
What's missing for it to be `#ifdef MODULE_ESP_RTC_TIMER`?

Reply:
The problem is that the RTC can't be used for alarms in active mode because the RTC main timer doesn't work in active mode. Therefore, the comment above this line.
https://github.com/RIOT-OS/RIOT/blob/f69db9df714293be19fc7de1d6aa895443670f3c/cpu/esp32/periph/rtc.c#L167-L173

A solution might be, to use the system timer for an alarm in active mode and to set the RTC alarm when going into deep sleep mode. But this isn't realized yet. But this is a todo. That's why the user has no possibility to enable module `esp_rtc` and that's the reason why it is still `#if 0`.

Reply:
An approach I could imagine to reduce the amount of printed information is to embedd these printings in `#if DEVELHELP´ so that they all can be activated with only one compile time switch instead of enabling them file by file with `ENABLE_DEBUG`. This could be done in a cleanup PR.